test/controllers/songs_controller_test.rb

```diff
@@ -1,19 +1,19 @@
 require 'test_helper'
 
 class SongsControllerTest < ActionController::TestCase
-  # test "should get index" do
-  #   get :index
-  #   assert_response :success
-  # end
+  test "should get index" do
+    get :index
+    assert_response :success
+  end
 
-  # test "should get new" do
-  #   get :new
-  #   assert_response :success
-  # end
-
-  # test "should get edit" do
-  #   get :edit
-  #   assert_response :success
-  # end
+  test "index should retrieve relevant search results" do
+    get :index, sSearch: 'hand', format: :json, xhr: true
+    songs = JSON.parse(@response.body)['data']
+    assert_equal(songs.length, 2)
```

Comment:
This test is too flaky because as we add new song fixtures to songs.yml, there might eventually be another one with "hand" in the lyrics and suddenly this search breaks. Instead, I would do two `assert_contains` to check that the two songs currently in there are in there next time.